create-custom-map-style/tileserver/setAwsEnvVariables.sh

```diff
@@ -0,0 +1,28 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: MIT-0
+#!/bin/bash
+
+export -n AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY AWS_SESSION_TOKEN
+creds_json=$(aws --output json sts get-session-token --duration-seconds 86400 --profile $1)
```

Comment:
Why? This is redundant unless the profile is different from the default profile.

Presumably you already have credentials configured (in `~/.aws/credentials`, which will also work here) in order to call `aws sts ...`.

Reply:
I don't normally use my default profile because I have dozens of AWS accounts I work in on a daily basis. I usually want to be very careful that I'm fetching credentials for the correct user. If these options aren't normally added to our documentation I can remove them no problem. I just personally find it helpful to specify the profile

Reply:
Sorry, I wasn't clear. I think including the profile is fine (same workflow here!), I just don't think you need to use STS to get credentials when you already have them (associated with whichever profile).

Reply:
All AWS SDKs should be able to use credentials in the same way, no matter where they come from, so setting `AWS_*` is likely unnecessary if `aws <whatever>` already works (and uses the same role).

Reply:
Ok I'm going to leave this part out then and add a pre-req to have these environment variables set correctly